frontend/src/services/api.ts

```diff
@@ -0,0 +1,51 @@
+import { User } from "../types/index.ts";
+
+let users: User[] = [
+  {
+    id: 1,
+    name: "John",
+    email: "john@example.com",
+    password: "password123",
+    role: "user",
+  },
+  {
+    id: 2,
+    name: "admin",
+    email: "admin@example.com",
+    password: "password456",
+    role: "admin",
+  },
+];
+
```

Comment:
separate this users variable into a `constants/` directory and make it a `const` instead of `let`